<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DamNGU Map Editor</string>
  </property>
  <widget class="QWidget" name="Editor">
   <widget class="QScrollArea" name="MapScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>871</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="MapScrollAreaContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>839</width>
       <height>869</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="MapArea">
        <property name="text">
         <string>Map</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SettingsBox">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>730</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QCheckBox" name="DisplayLayersCheckbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Display both layers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="MapWidthLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Map width</string>
     </property>
    </widget>
    <widget class="QLabel" name="MapHeightLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Map height</string>
     </property>
    </widget>
    <widget class="QLabel" name="PageWidthLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Page width</string>
     </property>
    </widget>
    <widget class="QLabel" name="PageHeightLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Page height</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MapWidth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="MapHeight">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PageWidth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PageHeight">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ActiveLayerBox">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>570</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Active Layer</string>
    </property>
    <widget class="QRadioButton" name="BackgroundRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Background</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="DecorationRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Decoration</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="FlagsRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Flags</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PaletteBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Palette</string>
    </property>
    <widget class="QLabel" name="Palette">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>512</height>
      </size>
     </property>
     <property name="text">
      <string>Palette</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LocationBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>560</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Location</string>
    </property>
    <widget class="QLabel" name="PageXLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Page X</string>
     </property>
    </widget>
    <widget class="QLabel" name="PageYLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Page Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="PageTileXLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Page Tile X</string>
     </property>
    </widget>
    <widget class="QLabel" name="PageTileYLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Page Tile Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="MapXLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Map X</string>
     </property>
    </widget>
    <widget class="QLabel" name="MapYLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Map Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MapX">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="PageX">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="PageTileX">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="MapY">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="PageY">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="PageTileY">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ImpassableStatusCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Impassable</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TeleportStatusCheck">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Teleport </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TBDStatusCheck">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TBD</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FlagsBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>730</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Set Flags</string>
    </property>
    <widget class="QCheckBox" name="ImpassableCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Impassable</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TeleportCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Teleport</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MapID">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="MapIDLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Map ID</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TBDCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TBD</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1450</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
